app/views/hyrax/my/collections/_modal_delete_collections_deny.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="modal fade" id="collection-to-delete-deny-modal" tabindex="-1" role="dialog" aria-labelledby="deny-delete-collection-label">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <form class="deny-delete-collection-form">
```

Comment:
It might not matter and not sure if this was existing code, but I'd check that this `<form>` is not embedded within another form for best practice.  Does the dismiss need to be wrapped in a `<form>`, or is it just styling?

Reply:
Looks like there are 5 modals for collections in the same directory that also use <form> in the same way.  @adamjarling If you have a suggestion for an alternative approach, can you open a refactor issue for all the modals to follow a recommended pattern?